homework-submission/week2/src/App.css

```diff
@@ -0,0 +1,264 @@
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+  font-weight: 200;
+}
+
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+html {
+  height: 100%;
+}
+
+body {
+  background: rgb(50, 50, 50);
+  color: whitesmoke;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1.2rem;
+}
+
+.todo-list {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.list-item {
+  color: red;
+  list-style: none;
+  text-align: center;
+  width: 275px;
+  margin: 25px;
+  min-height: 150px;
+  padding-top: 5px;
+  padding-bottom: 10px;
+  position: relative;
+  border: 1px solid rgb(255, 215, 7);
+  border-top: 60px solid rgb(255, 215, 7);
+  font-family: 'Reenie Beanie';
```

Comment:
what in the world is reenie beenie

Reply:
@nick-barth :sweat_smile:  isn't that cool?